frontend/src/metabase-lib/fields.ts

```diff
@@ -72,3 +73,22 @@ export function legacyRef(
 ): FieldReference {
   return ML.legacy_ref(column);
 }
+
+/**
+ * Info about FieldValues/remapping for the purposes of powering search widgets in filter modals.
+ */
+function fieldValuesSearchInfo(
```

Comment:
I decided to call this `fieldValuesSearchInfo` and not just `fieldValuesInfo`, because these three little bits of info are hardly the only info you might have about field values or remapping.